src/test/java/com/mfarag/todo/TaskListTest.java

```diff
@@ -2,12 +2,35 @@
 
 import org.junit.Test;
 
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 
 public class TaskListTest {
 
     @Test
     public void TestReturnNoTasksIfNoneExists() {
-        assertNull(new TaskList().getTask());
+
+        assertNull(new TaskList().getTask(0));
+    }
+
+    @Test
+    public void TestReturnTaskIfTaskExists() {
+        TaskList taskList = new TaskList() ;
+        taskList.addTask("New Task");
+        assertNotNull(taskList.getTask(0));
+    }
+
+    @Test
+    public void TestReturnFirstTask() {
+        TaskList taskList = new TaskList() ;
+        taskList.addTask("New Task");
```

Comment:
You need to add another task with different string to make sure you are really getting the first task.